substrate/utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -116,6 +117,21 @@ pub struct StorageParams {
 	/// Include child trees in benchmark.
 	#[arg(long)]
 	pub include_child_trees: bool,
+
+	/// Enable PoV recorder.
+	///
+	/// The recorder has impact on performance when benchmarking with the TrieCache enabled.
+	/// It should be used for the final results for parachains.
+	#[arg(long)]
+	pub enable_pov_recorder: bool,
```

Comment:
When would we ever disable it? When benchmarking for a relay/solo chain?

Reply:
Yeah I think this should be in the doc comment. To advise users when to set the recorder. Maybe default should even be to set it to true.